app/views/includes/organisation-overview.njk

```diff
@@ -36,16 +36,16 @@
               <td headers="grade om">{{ member.gradeCode }}</td>
             {% endif %}
             {% if member.capacityPercentage > 110 %}
-              <td class="capacity-red" headers="current cap">{{ member.capacityPercentage | round(1) }}%<span class="govuk-visually-hidden">. This is over 110% capacity</span></td>
+              <td data-order="{{ member.capacityPercentage }}" class="capacity-red" headers="current cap">{{ member.capacityPercentage | round(1) }}%<span class="govuk-visually-hidden">. This is over 110% capacity</span></td>
             {% else %}
-              <td headers="current cap">{{ member.capacityPercentage | round(1) }}%</td>
+              <td data-order="{{ member.capacityPercentage }}" headers="current cap">{{ member.capacityPercentage | round(1) }}%</td>
             {% endif %}
             <td headers="points cap">{{ member.availablePoints }}</td>
             <td headers="totalPoints cap">{{ member.totalPoints }}</td>
             {% if member.capacityPercentage > 110 %}
-              <td class="capacity-red" headers="remaining cap">{{ member.remainingPoints }}<span class="govuk-visually-hidden">. This is over 110% capacity</span></td>
+              <td data-order="{{ member.remainingPoints }}" class="capacity-red" headers="remaining cap">{{ member.remainingPoints }}<span class="govuk-visually-hidden">. This is over 110% capacity</span></td>
```

Comment:
I believe the ordering was incorrect due to the hidden content, which made the JS treat it as a string